tests/CI/test_quantization.py

```diff
@@ -421,8 +405,27 @@ def test_quantization_for_llm(self):
                                                      device_map="cpu"
                                                 )
         output = bit8_model(dummy_input)
-        self.assertTrue(isclose(float(output[0][0][0][0]), -7.2695, rel_tol=1e-04))
-
+        self.assertTrue(isclose(float(output[0][0][0][0]), 0.1674591302871704, rel_tol=1e-04))
+
+        #GPTQ
```

Comment:
could we also add awq test?

Reply:
AWQ, We have supported it before，but it is generally replace all linear from torch.nn.Linear to QuantizedLinearQbits